projects/core/src/window/window-ref.ts

```diff
@@ -1,25 +1,28 @@
 import { DOCUMENT, isPlatformBrowser } from '@angular/common';
 import { Inject, Injectable, PLATFORM_ID } from '@angular/core';
+import { SERVER_REQUEST_ORIGIN, SERVER_REQUEST_URL } from '@spartacus/core';
```

Comment:
[medium] Circular dep on `'@spartacus/core';`. It will compile with warnings
```suggestion
import { SERVER_REQUEST_ORIGIN, SERVER_REQUEST_URL } from '../util/ssr.tokens';
```